nuget/rlclientlib.nuspec.in

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<package>
+  <metadata>
+    <id>RLClientLibNative-v@MSVC_TOOLSET_VERSION@-@NATIVE_NUGET_PLATFORM_TAG@</id>
+    <version>1.0.0</version>
+    <title>Reinforcement Learning Client Library Native</title>
+    <authors>John Langford et al</authors>
+    <description>RLClientLib is an Interaction-side integration library for Reinforcement Learning loops: Predict, Log, Learn, Update</description>
+    <copyright>Copyright (c) by respective owners including Yahoo!, Microsoft, and individual contributors.</copyright>
+    <tags>native</tags>
+  </metadata>
```

Comment:
Should specify license as MIT. Needed for component governance.

Reply:
added